pkg/processor/procbuiltin/parsejson.go

```diff
@@ -54,6 +54,11 @@ func parseJSON(
 		switch data.(type) {
 		case record.RawData:
 			var jsonData record.StructuredData
```

Comment:
Nitpick: In the function's signature there's `config` which is unused. Credits go to GoLand.

Reply:
IMHO it's OK if it's left unused in those 2 methods (we don't _have_ to use it if we don't need a configuration), but where we don't need a parameter, we shouldn't put it. But again, really a nitpick.